dev/test/specs/starter-test.js

```diff
@@ -303,7 +308,7 @@ function _waitForAndMaybeAcceptSuggestion(textSubstring, accept) {
           return false;
         }
 
-        console.log(`found: desiredSuggestion`);
+        console.log(`found: desiredSuggestion ${desiredSuggestion}`);
```

Comment:
Suggest surrounding the value with '' so that eng debugging can see clearly in case the value reads confusingly.